### ISSUE TITLE
Unsupervised VOS Setting

### ISSUE THREAD

**Issue by Spritea** [henghuiding/MOSE-api#16]
Thanks for your answer before!

I have another question about the unsupervised VOS setting. Since I want to test some VIS methods on MOSE and need to follow the same way you used in the MOSE paper. From my understanding, unsupervised VOS with multiple objects = VIS (VIdeo Instance Segmentation), both segment and track objects of predefined categories without reference on the first frame, is that right?

Therefore, when you run those methods listed in Table 5, you train these methods on the training set of MOSE (only on those videos where the first frame is exhausitively labeled), and then test them on the validation set (only those videos with exhasutive labeled first frame) without reference on the first frame, which is the same way of trainging and testing VIS method, right?

Could you please clarify this? Thanks a lot!
